java/org/contikios/cooja/GUI.java

```diff
@@ -173,7 +173,7 @@ public void endDraggingFrame(JComponent f) {
       }
     });
     // Dragging windows on OS X leaves residue from the borders with FlatLaf, so avoid setting dragMode.
-    if (cooja.configuration.lookAndFeel() != LookAndFeel.FlatLaf) {
+    if (Cooja.configuration.lookAndFeel() != LookAndFeel.FlatLaf) {
```

Comment:
I was aware of a few of these warnings but left them because I was hoping the static member would eventually become non-static. But that's not happening anytime soon, so might as well do this.